### PR TITLE
And more changes triggered by translation work.

### PR DESCRIPTION
**Pull request by smoe**

### CODE REVIEW

File: docs/src/gcode/overview.adoc [hunk 1]
```diff
@@ -968,8 +968,7 @@ control.
 [[gcode:comments]]
 == Comments(((Comments)))
 
-Comments are purely informative and have no
-influence on machine behaviour
+Comments are purely informative and have no influence on machine behaviour.
```
Comment (petterreinholdtsen) [inline docs/src/gcode/overview.adoc:971]: What is the purpose of this change?  It look like it should have no effect on asciidoc nor po4a.

**Comment by hansu** [comment]
@smoe, do you still have https://github.com/LinuxCNC/linuxcnc/pull/1892#issuecomment-1208977409 in mind?

File: docs/src/gcode/overview.adoc (hunk 1 above)
Comment (smoe) [reply docs/src/gcode/overview.adoc:971]: Asciidoc eye candy. Important to me when I see it.

**Comment by smoe** [comment]
> @smoe, do you still have [#1892 (comment)](https://github.com/LinuxCNC/linuxcnc/pull/1892#issuecomment-1208977409) in mind?

I have separate branches for every machine that I am working on. That other rebase needs to wait just a bit :o/

File: docs/src/hal/halmodule.adoc [hunk 2]
```diff
@@ -213,7 +213,7 @@ hal.connect("pinname","signal_name")
 
 === disconnect
 
-Disconnect a pin from a signal. +
+Disonnect a pin from a signal.
```
Comment (hansu) [inline docs/src/hal/halmodule.adoc:216]: You inserted a typo :)

File: docs/src/gcode/overview.adoc (hunk 1 above)
Comment (hansu) [reply docs/src/gcode/overview.adoc:971]: Will that be a blocker for this PR? :smile:  :innocent:

**Comment by petterreinholdtsen** [comment]
[Hans]
> Will that be a blocker for this PR? :smile:  :innocent:

Not for me, at least.  I just wondered if there was something I did not
know about asciidoc and po4a.

-- 
Happy hacking
Petter Reinholdtsen

**Comment by smoe** [comment]
The atom editor triggers unintended changes with me. Should stop using it. Hoping to have fixed that build error.
